apps/api/urls.py

```diff
@@ -1,9 +1,9 @@
 from django.urls import path
 
-from api.views import calculatecoordinate
+from apps.api.views import calculatecoordinate, crawler
```

Comment:
you don't need `crawler` import anymore.

Reply:
Opps

Reply:
fixed on https://github.com/sosbrumadinho/brumadinho_location/pull/68/commits/eaab178b07732a46f48f882691bb20ead7c3c3c6